save-cloud-common/src/jvmMain/kotlin/com/saveourtool/save/entities/cosv/RawCosvFile.kt

```diff
@@ -34,7 +31,7 @@ class RawCosvFile(
     @ManyToOne
     @JoinColumn(name = "organization_id")
     var organization: Organization,
-    @Enumerated
+    @Enumerated(EnumType.STRING)
```

Comment:
old files were not removed, because of this
but how they were **correctly** uploaded with incorrect type?

Reply:
we have event on database level which removes stale entities in `raw_cosv_file` with status = `PROCESSED`